pkg/controller/istiocontrolplane/istiocontrolplane_controller.go

```diff
@@ -73,13 +72,13 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 
 	// TODO(user): Modify this to be the types you create that are owned by the primary resource
 	// Watch for changes to secondary resource Pods and requeue the owner IstioControlPlane
-	err = c.Watch(&source.Kind{Type: &corev1.Pod{}}, &handler.EnqueueRequestForOwner{
-		IsController: true,
-		OwnerType:    &v1alpha2.IstioControlPlane{},
-	})
-	if err != nil {
-		return err
-	}
+	//err = c.Watch(&source.Kind{Type: &corev1.Pod{}}, &handler.EnqueueRequestForOwner{
+	//        IsController: true,
+	//        OwnerType:    &v1alpha2.IstioControlPlane{},
+	//})
+	//if err != nil {
+	//        return err
+	//}
```

Comment:
I think you can just remove this. I don't see a use case for us wanting to watch Pods.

Reply:
removed this section.